### PR TITLE
add warnings to L1TGlobalPrescaler (93x)

### PR DESCRIPTION
**Pull request by fwyzard**
- warn if `l1tPrescales` has the wrong number of elements
  - warn if the target prescales are lower than the original ones

### CODE REVIEW

**Comment by fwyzard** [comment]
@cmsbuild, please test

**Comment by fwyzard** [comment]
bugfix

**Comment by fwyzard** [comment]
@cmsbuild, please test

**Comment by rekovic** [comment]
+1

File: L1Trigger/L1TGlobal/plugins/L1TGlobalPrescaler.cc [hunk 1]
```diff
@@ -100,18 +99,38 @@ namespace {
...
 #include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
 #include "FWCore/Utilities/interface/EDMException.h"
 #include "CondFormats/DataRecord/interface/L1TGlobalPrescalesVetosRcd.h"
 #include "CondFormats/L1TObjects/interface/L1TGlobalPrescalesVetos.h"
 #include "DataFormats/L1TGlobal/interface/GlobalAlgBlk.h"
 
+
+namespace {
+
+  template <typename T, std::size_t N, typename S>
```
Comment (fwyzard) [inline L1Trigger/L1TGlobal/plugins/L1TGlobalPrescaler.cc]: @Dr15Jones @wddgit can you think of any way to support
```c++
template <typename T, std::size_t N>
std::array<T, N>
ParameterSet::getParameter<std::array<T, N>>(...)
```
directly in `ParameterSet` ?

I I understand that it cannot be done as an overload or partial specialisation of `ParameterSet::getParameter<T>(...)` .

**Comment by davidlange6** [comment]
+1

**Pull request merged**